### DISCUSSION TITLE
How to prevent obsidian markdown link to be converted

### DISCUSSION THREAD

**Discussion by weihong0827** [ObsidianPublisher/obsidian-github-publisher#223] [Q&A — Plugin] [answered]
I just want to upload my raw obsidian notes to my GitHub since my publish platform supports the use of wiki links but it seems like even if I didn't turn on any link conversion options, it still convert ultimately when it is uploaded to github

my publisher config
<img width="783" alt="image" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/48542562/054d61a9-4110-4397-9bac-40dd9f1586d4">

what is shown on GitHub
<img width="210" alt="image" src="https://github.com/ObsidianPublisher/obsidian-github-publisher/assets/48542562/bb142f3f-2fd3-4dcd-b03e-d799241ed54f">

**Comment by Mara-Li** [comment, marked as answer]
Click on the button for texte replacer et remove the regex
